agent/src/iconfucius/client/server.py

```
@@ -0,0 +1,1053 @@
+        for b in tool_blocks:
+            if b.name == "trade_sell":
+                continue
+            usd = b.input.get("amount_usd")
+            if usd is not None and not b.input.get("amount"):
+                try:
+                    rate = get_btc_to_usd_rate()
+                    sats = int((usd / rate) * 100_000_000)
+                    b.input["amount"] = sats
+                    del b.input["amount_usd"]
+                except Exception:
+                    pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)